cpp/src/main.cpp

```diff
@@ -0,0 +1,58 @@
+#include <boost/program_options.hpp>
+#include <iostream>
+
+#include "migrate/migrate.hpp"
+#include "sensor/sensor.hpp"
+#include "server/server.hpp"
+
+namespace po = boost::program_options;
+
+int main(int argc, char* argv[]) {
+    po::options_description desc("Allowed options");
+    // clang-format off
```

Comment:
why ?

Reply:
I get it, but TBH, I am not fan of having formatter and then disable it for part of the code just because you don't like results, you either have it or don't.
Again, just observation, not asking to fix it.

Reply:
Maybe there is some clang-format config to fix it, idk.